src/sql/parts/notebook/cellViews/outputArea.css

```diff
@@ -2,13 +2,14 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-
-code-cell-component {
+output-area-component {
 	display: block;
 }
 
-code-cell-component .notebook-output {
+output-area-component .notebook-output {
```

Comment:
Thanks - really like this change overall. It puts the display logic in the right place.